Sources/apm-agent-ios/Agent.swift

```diff
@@ -31,35 +31,38 @@ public class Agent {
     
     private static var instance: Agent?
 
-    var configuration: AgentConfiguration
+//    var configuration: AgentConfiguration
```

Comment:
Is this commented on purpose?

Reply:
yes, configuration was moved into the config manager. I'll clean this up.